dev/integration_tests/ios_platform_view_tests/test_driver/platform_view_e2e.dart

```diff
@@ -0,0 +1,59 @@
+// Copyright 2014 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'package:flutter/material.dart';
+import 'package:flutter/widgets.dart';
+import 'package:flutter_test/flutter_test.dart';
+import 'package:e2e/e2e.dart';
+
+void main() {
```

Comment:
nit: let's add a note that it is not currently running and link to an issue.

Reply:
done